curvefs/src/mds/common/storage_key.h

```diff
@@ -39,6 +39,9 @@ const char FS_NAME_KEY_END[] = "fs_02";
 
 const char FS_ID_KEY_PREFIX[] = "fs_02";
 
+const char CHUNKID_NAME_KEY_PREFIX[] = "chunkid_01";
+const char CHUNKID_NAME_KEY_END[] = "chunkid_02";
```

Comment:
for consistency, it's better to define it as "fs_03" and "fs_04"

Reply:
fix